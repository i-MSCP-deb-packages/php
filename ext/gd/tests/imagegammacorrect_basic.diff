001+ d2590dddbe158194bc41f800dd0c664e
001- 30639772903913594bc665743e1b9ab8