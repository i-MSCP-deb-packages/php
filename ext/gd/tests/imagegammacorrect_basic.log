
---- EXPECTED OUTPUT
30639772903913594bc665743e1b9ab8
---- ACTUAL OUTPUT
d2590dddbe158194bc41f800dd0c664e
---- FAILED
