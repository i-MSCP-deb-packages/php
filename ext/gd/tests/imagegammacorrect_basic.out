d2590dddbe158194bc41f800dd0c664e